### ISSUE TITLE
R requirements

### ISSUE THREAD

**Issue by yeesian** [joehuchette/OR-software-tools-2015#1]
@gutin tried to follow the pre-assignment instructions for `2-intermediate-R`, and got the following:

```
> library(sqldf)
Loading required package: RSQLite.extfuns
Error: package ‘RSQLite.extfuns’ could not be loaded
In addition: Warning message:
In library(pkg, character.only = TRUE, logical.return = TRUE, lib.loc = lib.loc) :
 there is no package called ‘RSQLite.extfuns’
```

and

```
Warning in install.packages :
 package ‘RSQLite.extfuns’ is not available (for R version 3.0.2)
```

Maybe we should warn about the (minimal) version of R, and how/where to get it?

**Comment by joehuchette** [comment]
Yes, we noticed yesterday that there was a version incompatibility with `sqldf`. Per [the documentation](https://code.google.com/p/sqldf/), you need R version ≥ 3.1.0. Version 3.1.2 can be downloaded on the [R website](http://www.r-project.org/) I'll add a note in the README.

**Comment by joehuchette** [comment]
Closed per d76373e3b6269e96b45f9238d427ad2600f40ed9
